src/System.CommandLine/Command.cs

```diff
@@ -135,11 +135,11 @@ public void Add(Symbol symbol)
         /// <summary>
         /// Parses an array strings using the command.
         /// </summary>
-        /// <param name="args">The string arguments to parse.</param>
+        /// <param name="arguments">The string arguments to parse.</param>
         /// <param name="configuration">The configuration on which the parser's grammar and behaviors are based.</param>
         /// <returns>A parse result describing the outcome of the parse operation.</returns>
-        public ParseResult Parse(string[]? args, CommandLineConfiguration? configuration = null)
-            => Parser.Parse(this, args, configuration);
+        public ParseResult Parse(IReadOnlyList<string> arguments, CommandLineConfiguration? configuration = null)
```

Comment:
I would suggest the name `args` here as it aligns with the naming convention for `Main(string[] args)`.

Reply:
> I would suggest the name args here as it aligns with the naming convention for Main(string[] args).

done